widgets.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
```

Comment:
We should move the HTML files into a folder called "examples" or something - I don't think they should be cluttering up the root directory

Reply:
I made this change in the `layout-persistence` branch